### PR TITLE
ENH bump the appveyor image to vs2017

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Added a ``news`` entry

### CODE REVIEW

**Comment by beckermr** [comment]
@isuruf This is ready for review

File: conda_smithy/configure_feedstock.py [hunk 1]
```diff
@@ -1060,6 +1060,7 @@ def _appveyor_specific_setup(jinja_env, forge_config, forge_dir, platform):
     build_setup = new_build_setup.strip()
 
     forge_config["build_setup"] = build_setup
+    forge_config["appveyor"] = {"image": "Visual Studio 2017"}
```
Comment (isuruf) [inline conda_smithy/configure_feedstock.py:1063]: Can this use the `_load_forge_config` function where this can be overriden?

Comment (beckermr) [reply conda_smithy/configure_feedstock.py:1063]: IDK where that is, but yeah maybe. I'll look.

Comment (beckermr) [reply conda_smithy/configure_feedstock.py]: done.

**Comment by beckermr** [comment]
Ready for another look @isuruf

**Pull request merged**